src/cryptography/hazmat/backends/utils.py

```diff
@@ -0,0 +1,38 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+
+class GcmSizeValidator(object):
+    """
+    GCM may only encrypt up to 2**39 - 256 bits of plaintext, so we
+    must track the number of bytes we see.
+    """
+    _PLAINTEXT_BIT_LIMIT = 2 ** 39 - 256
+    _AAD_BIT_LIMIT = 2 ** 64 - 1
```

Comment:
I'd prefer these numbers to be bytes. The AAD bit limit isn't 0 mod 8 so it can't be perfectly represented without bits, but our API requires data be passed as bytes so it'd be reasonable to say it has to be > ((2 ** 64) / 8))